### ISSUE TITLE
Some problem converting Unity Save

### ISSUE THREAD

**Issue by ik9ro** [linuslin0/ACST#86]
Hi ! like in issue #80 that you said the tool supports only saves from legit game copy, I tried to use 3DM convert tool `Cracked -> Legit` but without sucess. And in the #80 you convert a non legit game copy to a legit one, can you tell what the steps for it ? 

Thanks in advance !

**Comment by ik9ro** [comment]
I was reading #84 and see that i need disable cloud save sync when i switch the saves, and worked for me ! cheers

**Comment by MAG1STO** [comment]
@ik9ro Please help me! I am also trying to convert a save using the 3DM convert tool, but the error appears "This CD KEY Can't Convert This Save File" Please describe the steps you are performing. Faced with such a mistake?

**Comment by ik9ro** [comment]
@MAG1STO I didn't faced this error specifically, but here are the steps i followed (it was on A.C Unity savegame):

```
1. Select my pirate saves from 3DM
2. Put them on save converter
3. Select "From 3DM Save"
4. Convert Save option
```

The error that I faced was my save is being replaced by the Ubisoft cloud save, and I managed to solve it by disabling cloud save in Ubisoft Connect options. 

I recommend you to try dowload any other save from the internet that works in the 3DM crack version, and then try to convert these for legit version. And see if it works, maybe the problem are in your savegame or in the crack which may be different.

I hope you get succeed !

**Comment by MAG1STO** [comment]
@ik9ro Thank you very much! I figured out the problem. It turns out that after using the program "AC IV: Black Flag Save Convert Tool By 3DM" files are created ".bak"and I thought that these files were converted saves for my licensed version until I decided to look on the Internet what the extension means".bak" :D

Thank you very much again! If it wasn't for you, I wouldn't be able to figure out this problem :)

If someone does not know how to disable the synchronization of cloud saves, then this is done in the Ubisoft Connect settings in the "General". I hope I was able to help someone
